net/pdnsd/files/pdnsd.init

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2019 OpenWrt.org
+
+USE_PROCD=1
+
+START=65
+STOP=10
+
+PROG=/usr/sbin/pdnsd
+CONFFILE=/etc/config/pdnsd.conf
+PIDFILE=/var/run/pdnsd.pid
+CACHEDIR=/tmp/pdnsd
+CACHEFILE=$CACHEDIR/pdnsd.cache
+
+start_service() {
+	gen_cache
+	procd_open_instance
+	procd_set_param command "$PROG" -c "$CONFFILE" -p "$PIDFILE"
+	procd_set_param limits nofile=4096
+	procd_set_param stdout 1
+	procd_set_param stderr 1
+	procd_set_param pidfile "$PIDFILE"
```

Comment:
is the pid file still important? procd doesn't need them, this ability to write a pidfile is only to help it with other services. so if this was just migrated from the original init script here, can we not just drop the pidfile?

Reply:
Yes, I remove pidfile, thank @karlp. :-)